src/pages/Credentials/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+
+import { createToken, deleteToken, listTokens } from '@/services/credentialsService';
+import './styles.scss';
+
+type CredentialsProps = {
+  workspaceProject: string;
+};
+
+export const Credentials = ({ workspaceProject }: CredentialsProps) => {
+  const [tokens, setTokens] = useState([]);
+
+  useEffect(() => {
+    fetchTokens();
+  }, []);
+
+  const fetchTokens = async () => {
+    try {
+      const fetchedTokens = await listTokens();
+      setTokens(fetchedTokens);
+    } catch (error) {
+      console.error('Failed to fetch tokens:', error);
+    }
+  };
+
+  const handleCreateToken = async () => {
+    try {
+      const newToken = await createToken();
+      setTokens([...tokens, newToken]);
+      alert(`Token created: ${newToken.token}\nThis is your only chance to copy it!`);
+    } catch (error) {
+      console.error('Failed to create token:', error);
+    }
+  };
+
+  const handleDeleteToken = async (tokenId) => {
+    try {
+      await deleteToken(tokenId);
+      setTokens(tokens.filter((token) => token.id !== tokenId));
+    } catch (error) {
+      console.error('Failed to delete token:', error);
+    }
+  };
+
+  return (
+    <div className="workspace__credentials">
+      <h2>API Credentials ({workspaceProject})</h2>
```

Comment:
Be careful using headers, I assume your using this as an `h2` is styled pretty much as you want. but really headings have a semantic meaning, see https://www.a11yproject.com/posts/how-to-accessible-heading-structure/. This project has no `h1` heading, so having `h2` and `h3` breaks accessibility. Headings should only be used to semantically structure HTML, never for styling, better to use a `p` tag and style it to what you need.